schwab/streaming.py

```diff
@@ -390,1254 +386,1231 @@ async def login(self, websocket_connect_args=None):
             await self._send({'requests': [request]})
             await self._await_response(request_id, 'ADMIN', 'LOGIN')
 
+    async def logout(self):
+        """
+        `Official Documentation <https://developer.schwab.com>`__
+        """
+        request, request_id = self._make_request(
+            service='ADMIN', command='LOGOUT',
+            parameters={})
+        async with self._lock:
+            await self._send({'requests': [request]})
+            await self._await_response(request_id, 'ADMIN', 'LOGOUT')
+
+
     ##########################################################################
     # ACCT_ACTIVITY
 
     class AccountActivityFields(_BaseFieldEnum):
-        '''
-        `Official documentation <https://developer.tdameritrade.com/content/
-        streaming-data#_Toc504640580>`__
+        """
+        `Official documentation <https://developer.schwab.com>`__
 
         Data fields for equity account activity. Primarily an implementation detail
         and not used in client code. Provided here as documentation for key
         values stored returned in the stream messages.
-        '''
+        """
 
-        #: Unknown
-        FIELD_0 = 0
+        # Passed back to the client from the request to identify a subscription this response belongs to.
+        SUBSCRIPTION_KEY = 0
 
-        #: Unknown
-        FIELD_1 = 1
+        # Account Number that the activity occurred on.
+        ACCOUNT = 1
 
-        #: Unknown
-        FIELD_2 = 2
+        # Message Type that dictates the format of the Message Data field.
+        MESSAGE_TYPE = 2
 
-        #: Unknown
-        FIELD_3 = 3
+        # The core data for the message. Either JSON-formatted data describing the update, NULL in some cases,
```

Comment:
I presume these documentation strings are copied from the [official documentation PDF](https://discord.com/channels/720378361880248621/1062730209943629915/1248312546935574578) that was shared in the Discord server?

Reply:
Yes, this is what I used.